proposals/0201-high-level-interface-overlay-controllers.md

```diff
@@ -196,7 +262,7 @@ This property will be handled just like `SDLAlertController` and maps to `Perfor
 
 #### `muteMediaSource`
 
-This property maps to `PerformAudioPassThru.muteAudio`. If this property is false the parameter `muteAudio` will be omitted.
+This property maps to `PerformAudioPassThru.muteAudio`. Some initializers of this overlay controller won't ask for this flag therefore by default it's set to YES. When starting this overlay controller the RPC request will always include the parameter `muteAudio` due to unspecific defvalue in the mobile API.
```

Comment:
Change to:
This property maps to `PerformAudioPassThru.muteAudio`. Some initializers of this overlay controller won't ask for this **flag,** therefore by default **its** set to YES. When starting this overlay controller the RPC request will always include the parameter `muteAudio` due to unspecific defvalue in the mobile API.